cmd/buildah/run.go

```diff
@@ -79,10 +79,12 @@ func runCmd(c *cli.Context) error {
 		Args:     c.StringSlice("runtime-flag"),
 	}
 
-	if c.Bool("tty") {
+	options.Terminal = buildah.DefaultTerminal
```

Comment:
This shouldn't be necessary, as `DefaultTerminal` is the zero value, which `options` should already include as a default.

Reply:
I'd not thought of that.  I'll remove that line, retest and will cut a new PR.